### PR TITLE
Adding "publish-to-gts" script

### PR DESCRIPTION
**Pull request by 77nnit**
New script to publish markdown notes to GoToSocial activitypub server.

### CODE REVIEW

File: publish-to-gts/info.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Publish to GtS",
```
Comment (pbek) [inline publish-to-gts/info.json:2]: Can you maybe please use "GoToSocial" instead of "GtS" as name?

File: publish-to-gts/publish-to-gts.qml [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+        "name": "Server instance",
+        "description": "Server instance you want to connect to (i.e.: example.org - no spaces, no protocol, no slashes)",
+        "type": "string",
+        "default": "example.org",
+        },
+        {
+        "identifier": "authCode",
+        "name": "Authentication Code",
+        "description": "Code returned by GtS after performing a successful authentication, if you paste it here you won't need to authenticate again until expiry",
+        "type": "string",
```
Comment (pbek) [inline publish-to-gts/publish-to-gts.qml:25]: Do you want to consider `string-secret` from #3150?

**Comment by pbek** [comment]
It would make sense if you rebase your local git repo to `master`, you are pushing all your past commits to this pull request again. 😉

File: publish-to-gts/publish-to-gts.qml (hunk 2 above)
Comment (pbek) [reply publish-to-gts/publish-to-gts.qml:25]: You need to update `minAppVersion` too if you use the `string-secret`.

File: publish-to-gts/info.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  "identifier": "publish-to-gts",
+  "script": "publish-to-gts.qml",
+  "authors": ["@77nnit"],
+  "platforms": ["linux"],
+  "version": "0.1.0",
+  "minAppVersion": "24.10.5",
+  "description" : "Publish To GtS lets you publish your notes to your GoToSocial account."
```
Comment (pbek) [inline publish-to-gts/info.json:9]: Maybe link the name `GoToSocial` to https://gotosocial.org/?

File: publish-to-gts/info.json (hunk 1 above)
Comment (77nnit) [reply publish-to-gts/info.json:2]: Sure, will do

File: publish-to-gts/publish-to-gts.qml (hunk 2 above)
Comment (77nnit) [reply publish-to-gts/publish-to-gts.qml:25]: Yes, the next iteration will be with string -secret.

File: publish-to-gts/info.json (hunk 3 above)
Comment (77nnit) [reply publish-to-gts/info.json:9]: Sure

**Comment by 77nnit** [comment]
> It would make sense if you rebase your local git repo to `master`, you are pushing all your past commits to this pull request again. 😉

I realized it as soon as I created the pull request. I'm sorry for all these inconveniences, but I'm slowly learning GIT mechanics... not ye used to them!

**Comment by pbek** [comment]
> Should I close this Pull request, adjust things and open a new one? Wouldn't it be cleaner?

No, not needed. I'll just squash the commits.

**Pull request merged**